assets/js/gutenberg-syndicated-post.js

```diff
@@ -23,40 +23,40 @@ if (
 		message = sprintf(
 			/* translators: 1) Source of content, 2) Distributor post type singular name. */
 			__(
-				'Distributed from %1$s. This %2$s is linked to the original. Edits to the original will update this version.',
+				'Distributed from %1$s. This %2$s is linked to the origin post. Edits to the origin post will update this remote version.',
```

Comment:
```suggestion
				'Distributed from %1$s. This %2$s is linked to the origin %2$s. Edits to the origin %2$s will update this remote version.',
```

This is so the correct post type is displayed, eg podcasts will read:

> Distributed from Distributor Test Site. This Podcast is linked to the origin Podcast. Edits to the origin Podcast will update this remote version.

There is a catch, the `dtGutenberg.postTypeSingular` uses the case for the singular post name label in `register_post_type()`, most commonly with an upper case first letter.

We could convert to lowercase before use but I am not sure how this would work with non-english language sites.

Reply:
I think I'm fine with casing being a spot off on this.  Are there other places throughout this PR that should similarly be updated @peterwilsoncc?

Reply:
@jeffpaul A few more spots, I'll push some commits so I can make sure the string is available in each location.